src/__tests__/__snapshots__/documenter.test.ts.snap

```diff
@@ -729,7 +729,7 @@ Each Drawer is an item in the drawers wrapper with the following properties:
 - \`iconSvg\` (React.ReactNode) - (Optional) Specifies the SVG of a custom icon. For more information, see [SVG icon guidelines](/components/icon/?tabId=api#slots)
 
 #### DrawerAriaLabels
-- \`drawerName\` (string) - Label for the drawer itself.
+- \`drawerName\` (string) - Label for the drawer itself, and for the drawer trigger button tooltip text.
```

Comment:
I think "label" so far meant "ARIA label", but now the newly added part reads like "label for the drawer button tooltip text", which makes the meaning of "label" no more clear. To make it clear that in the case of the drawer trigger it is used as an ARIA label, and in the case of the tooltip it is used for the text, what about something like

> Label for the drawer itself. Also used for the trigger button tooltip text.

Reply:
I'll follow up on this